### PR TITLE
add is_in_division method for user model

### PR DESCRIPTION
**Pull request by LucasBertrand**

### CODE REVIEW

File: league/templates/league/players.html [hunk 1]
```diff
@@ -67,7 +67,7 @@
           {% for division in divisions %}
             <div class="checkbox div-checkbox ">
               <label>
-                <input class ="check-div"  onchange="filterdivs()" value="{{division}}" type="checkbox" checked>{{division.name}}
+                <input class ="check-div"  onchange="filterdivs()" value="{{division.name}}" type="checkbox" checked>{{division.name}}
```
Comment (climu) [inline league/templates/league/players.html:70]: This is an unrelated fix right?

Comment (LucasBertrand) [reply league/templates/league/players.html:70]: yep, the Datatable filtering didn't work because of that

Comment (LucasBertrand) [reply league/templates/league/players.html:70]: I though it was not worth to mention

**Comment by climu** [comment]
We have automated pylint test at travis and this red cross show that those failed: 
image
You can click those cross to have details: 
In this case:
```
************* Module league.models

league/models.py:808:0: C0303: Trailing whitespace (trailing-whitespace)

************* Module league.views

league/views.py:273:43: C0303: Trailing whitespace (trailing-whitespace)
```
You can also run pylint locally before pushing ;)

File: league/templates/league/players.html (hunk 1 above)
Comment (climu) [reply league/templates/league/players.html:70]: Nice catch!

Comment (LucasBertrand) [reply league/templates/league/players.html:70]: And i just see i forgot the top one ^^

**Comment by climu** [comment]
Just tagging #437 as this should fix it.
